src/services/coingecko/api/price.service.ts

```diff
@@ -10,11 +10,25 @@ export type Price = { [fiat: string]: number };
 export type PriceResponse = { [id: string]: Price };
 export type TokenPrices = { [address: string]: Price };
 
+const nativeAssetIdMap = {
+  '1': 'ethereum',
+  '42': 'ethereum',
+  '137': 'matic-network'
+};
+
+const platformIdMap = {
+  '1': 'ethereum',
+  '42': 'ethereum',
+  '137': 'polygon-pos'
+};
```

Comment:
It seems that `nativeAssetIdMap` and `platformIdMap` is defined twice? could we reuse it?

Reply:
Yeah, we could reuse this. The duplication is a little bit of laziness from me as I assume we're going to be deleting one of these services as we finish the migration.